source/schemas/types.txt

```diff
@@ -192,13 +198,23 @@ A ``bool`` is either ``true`` or ``false``.
 Mixed
 ~~~~~
 
-A ``mixed`` field may contain any schema type except for :ref:`arrays
-<schema-type-array>`, :ref:`embedded objects <schema-type-object>`,
-:ref:`sets <schema-type-set>`, and :ref:`dictionaries
-<schema-type-dictionary>`. App Services does not enforce a consistent type across
+.. include:: /includes/note-collections-in-mixed-sync.rst
+
+A ``mixed`` field may contain any schema type except for :ref:`embedded objects
```

Comment:
not sure if this has been noted elsewhere but mixed type also does _not_ support sets

Reply:
👍 it's mentioned in the next line